src/prefs.c

```diff
@@ -1108,6 +1108,9 @@ on_prefs_dialog_response(GtkDialog *dialog, gint response, gpointer user_data)
 
 		widget = ui_lookup_widget(ui_widgets.prefs_dialog, "check_symbol_auto_completion");
 		editor_prefs.auto_complete_symbols = gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(widget));
+		
+		widget = ui_lookup_widget(ui_widgets.prefs_dialog, "check_always_auto_close_brackets");
+		editor_prefs.always_auto_close_brackets = gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(widget));
```

Comment:
the setting is never loaded.  Also, new settings should use Stash API, and here probably be in *keyfile.c:init_pref_groups()*.